python-frontend/src/main/java/org/sonar/python/index/DescriptorUtils.java

```diff
@@ -67,11 +67,57 @@ public static Symbol symbolFromDescriptor(Descriptor descriptor, ProjectLevelSym
           .map(a -> DescriptorUtils.symbolFromDescriptor(a, projectLevelSymbolTable, symbolName, createdSymbolsByDescriptor, new HashMap<>()))
           .collect(Collectors.toSet()));
         return ambiguousSymbol;
+      case ALIAS:
+        Descriptor recreatedDescriptor = recreateDescriptorFromAlias((AliasDescriptor) descriptor);
+        return symbolFromDescriptor(recreatedDescriptor, projectLevelSymbolTable, symbolName, createdSymbolsByDescriptor, createdSymbolsByFqn);
       default:
         throw new IllegalStateException(String.format("Error while creating a Symbol from a Descriptor: Unexpected descriptor kind: %s", descriptor.kind()));
     }
   }
 
+  private static Descriptor recreateDescriptorFromAlias(AliasDescriptor aliasDescriptor) {
```

Comment:
I think there are no unit tests for this function, especially in the case that the original descriptor is not a class or a function.